www/js/plugin/storage.ts

```diff
@@ -145,25 +143,21 @@ function findMissing(fromKeys: any[], toKeys: any[]) {
 }
 
 export function storageSyncLocalAndNative() {
-  console.log('STORAGE_PLUGIN: Called syncAllWebAndNativeValues ');
+  logDebug('STORAGE_PLUGIN: Called syncAllWebAndNativeValues');
```

Comment:
TODO potential regression
Need to look into this. This function `storageSyncLocalAndNative` doesn't appear to be called anywhere.

`getUnifiedValue` is working and is validated by testing. But `storageSyncLocalAndNative` is not.

This would suggest we are only syncing values for a particular key when it is being requested.
We aren't regularly syncing all keys on each app startup / resume, [as was the pre-migration behavior](https://github.com/e-mission/e-mission-phone/commit/7e27ecbf0d7d923780ff348273cb94696e78fa6a)